cmd/snapctl/plugin.go

```diff
@@ -116,6 +116,88 @@ func unloadPlugin(ctx *cli.Context) {
 	fmt.Printf("Type: %s\n", r.Type)
 }
 
+func swapPlugins(ctx *cli.Context) {
+	// plugin to load
+	pAsc := ctx.String("plugin-asc")
+	var paths []string
+	if len(ctx.Args()) < 1 || len(ctx.Args()) > 2 {
```

Comment:
@tiffanyfj does `Args` return just positional arguments, or all items passed at the command line?  If so, wouldn't a swap with the loaded plugin being loaded with a signing file have a `len` of 4?
```
                      [1]        [2]          [3]            [4]
$ snapctl plugin swap ~/mock2 --plugin-asc ~/mock2.asc collector:mock:1
```